index.html

```diff
@@ -0,0 +1,168 @@
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="about-me">
+            <h2>About Me</h2>
+            <p> I can help you build a product, feature or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded,
+                don't hestiate to contact me.</p>
+            <div class="resume-btn"><a href="#">Get My Resume</a></div>
```

Comment:
Great job on your progress so far! Here are a few suggestions to further improve your code:

1. Button Design: As mentioned earlier, it's important to make sure your button matches the design. Currently, it seems that there are some discrepancies between the button styles in your code and the design. Please ensure that you update your code to make the button appearance consistent with the design.

2. Layout with Margin, Padding, and Font Styles: Instead of relying on fixed width and height, I recommend using appropriate margin, padding, and font styles to layout your elements. This will make your layout more flexible and responsive across different screen sizes.

3. Semantic Structure: In the "resume-btn" div, I suggest making it a wrapper and applying all button-specific styles to the "a" tag. This will provide a more semantic structure to your code and improve its readability and maintainability.

By implementing these suggestions, you will align your code more closely with the design and create a better user experience. Keep up the great work! 👍